specificatie/gba-openapi.yaml

```diff
@@ -0,0 +1,98 @@
+        '400':
+          $ref: 'common.yaml#/components/responses/400'
+        '401':
+          $ref: 'common.yaml#/components/responses/401'
+        '403':
+          $ref: 'common.yaml#/components/responses/403'
+        '404':
+          $ref: 'common.yaml#/components/responses/404'
+        '406':
+          $ref: 'common.yaml#/components/responses/406'
+        '429':
+          $ref: 'common.yaml#/components/responses/429'
+        '500':
+          $ref: 'common.yaml#/components/responses/500'
+        '501':
+          $ref: 'common.yaml#/components/responses/501'
+        '503':
+          $ref: 'common.yaml#/components/responses/503'
+        'default':
+          $ref: 'common.yaml#/components/responses/default'
+      tags:
+      - Reisdocumenten
+components:
+  schemas:
+    GbaReisdocument:
+      type: object
+      description: |
+                    Een document dat nodig is voor reizen naar het buitenland.
+                    * **soortReisdocument** : geeft aan welk Nederlands reisdocument is verstrekt. Omschrijving wordt gevuld op basis van de waarden die voorkomen in de tabel 'Reisdocument' (landelijke tabel 48) uit de Haal-Centraal-BRP-tabellen-bevragen API.
+                    * **autoriteitAfgifte** : geeft aan welke autoriteit het reisdocument heeft verstrekt. Omschrijving wordt gevuld op basis van de waarden die voorkomen in de tabel 'Autoriteit_Afgifte_Reisdocument' (landelijke tabel 49) uit de Haal-Centraal-BRP-tabellen-bevragen API.
+                    * **datumUitgifte** : de datum waarop het reisdocument is uitgegeven.
+                    * **datumEindeGeldigheid** : de datum waarop het reisdocument niet meer geldig is.
+                    * **datumInhoudingOfVermissing**: de datum waarop het reisdocument is vermist, ingehouden, ingeleverd, of van rechtswege is vervallen.
+                    * **aanduidingInhoudingOfVermissing** : Omschrijving wordt gevuld op basis van de waarden die voorkomen in de tabel 'Inhouding_Vermissing_Reisdocument' uit de Haal-Centraal-BRP-tabellen-bevragen API.
+      properties:
+        reisdocumentnummer:
+          $ref: 'reisdocument.yaml#/components/schemas/Reisdocumentnummer'
+        soortReisdocument:
+          $ref: 'common.yaml#/components/schemas/Waardetabel'
+        autoriteitAfgifte: 
+          $ref: 'common.yaml#/components/schemas/Waardetabel'
+        datumUitgifte:
+          $ref: 'datum.yaml#/components/schemas/GbaDatum'
+        datumEindeGeldigheid:
+          $ref: 'datum.yaml#/components/schemas/GbaDatum'
+        datumInhoudingOfVermissing:
+          $ref: 'datum.yaml#/components/schemas/GbaDatum'
+        aanduidingInhoudingOfVermissing:
+          $ref: 'common.yaml#/components/schemas/Waardetabel'
+        burgerservicenummer:
+          $ref: 'reisdocument.yaml#/components/schemas/Burgerservicenummer'
+        inOnderzoek:
+          $ref: 'in-onderzoek.yaml#/components/schemas/GbaInOnderzoek'
```

Comment:
Is in onderzoek nog wel van toepassing nu er sprake is van een andere bron dan de BRP?